prime-router/metadata/fhir_mapping/hl7/resource/DiagnosticReport.yml

```diff
@@ -14,35 +14,68 @@ identifier_1:
   constants:
     sys: "urn:id:extID"
 
-identifier_2:
+identifier_2a:
   condition: $valueIn NOT_NULL
   valueOf: datatype/Identifier_var
   generateList: true
+  useGroup: true
   expressionType: resource
   vars:
-    valueIn: ORC.3.1 | OBR.3.1
-    systemCX: ORC.3.2 | OBR.3.2
-    universalId: ORC.3.3 | OBR.3.3
-    universalIdType: ORC.3.4 | OBR.3.4
+    valueIn: ORC.3.1
+    systemCX: ORC.3.2
+    universalId: ORC.3.3
+    universalIdType: ORC.3.4
   constants:
     system: "http://terminology.hl7.org/CodeSystem/v2-0203"
     code: "FILL"
     display: "Filler Identifier"
 
-identifier_3:
+identifier_2b:
+  condition: $valueInORC NULL && $valueIn NOT_NULL
+  valueOf: datatype/Identifier_var
+  generateList: true
+  useGroup: true
+  expressionType: resource
+  vars:
+    valueIn: OBR.3.1
+    systemCX: OBR.3.2
+    universalId: OBR.3.3
+    universalIdType: OBR.3.4
+    valueInORC: ORC.3.1
+  constants:
+    system: "http://terminology.hl7.org/CodeSystem/v2-0203"
+    code: "FILL"
```

Comment:
Is there a reason the display is not added to 2b as well?

Reply:
It wasn't in the mappings for OBR, so I didn't include it for OBR, but I didn't want to edit ORC since there was a separate ticket for that. Probably could've left it in, but I don't see any justification for it in the documentation, and it doesn't seem to be used at all in the FHIR->HL7v2 translation.